.github/workflows/create-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Create Release
+on:
+  release:
+    types: [created]
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+
+      - name: Publish plugin
+        uses: eskatos/gradle-command-action@v1.2.0
+        env:
+          GRADLE_PUBLISH_KEY: ${{ secrets.GRADLE_PUBLISH_KEY }}
+          GRADLE_PUBLISH_SECRET: ${{ secrets.GRADLE_PUBLISH_SECRET }}
+        with:
+          arguments: publishPlugins -Pgradle.publish.key=${GRADLE_PUBLISH_KEY} -Pgradle.publish.secret=${GRADLE_PUBLISH_SECRET}
+
+      - name: Publish Javadocs
+        uses: eskatos/gradle-command-action@v1.2.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          arguments: gitPublishPush -Dorg.ajoberstar.grgit.auth.username=${GITHUB_TOKEN}
```

Comment:
Same comment here about sensitive data passing as arguments.

Reply:
I tested `org.gradle.project.org.ajoberstar.grgit.auth.username=somevalue` in my local, tried to print the system property `org.ajoberstar.grgit.auth.username`. It doesn't work as expected, it shows `null`, different from the result of 
```
./gradlew -Dorg.ajoberstar.grgit.auth.username=somevalue
```

Then I refer to https://github.com/ajoberstar/gradle-git-publish#migrating-from-orgajoberstargithub-pages to use environment variable `GRGIT_USER`.

Reply:
Hi @JLLeitschuh, could you help check if this makes sense to you?

Reply:
Can you use the environment variable?

Reply:
Yeah, I push a new commit to use `GRGIT_USER` in the publish plugin step